src/content/contributing/release-process/_index.en.md

```diff
@@ -0,0 +1,146 @@
+git push upstream v0.2.0-rc0
+```
+
+A tagged release should appear [here](https://github.com/submariner-io/lighthouse/tags)
+
+> https://github.com/submariner-io/lighthouse/tags
+
+A build for v0.2.0-rc0 should start and appear under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/lighthouse/branches)
+
+> https://travis-ci.com/github/submariner-io/lighthouse/branches
+
+For this example the build can be found [here](https://travis-ci.com/github/submariner-io/lighthouse/builds/153946391).
+
+Look under the "Active branches" section for v0.2.0-rc0, and monitor the build.
+Verify that the build successfully completes as indicated by a green checkmark at the right. At this point the images are pushed to quay.io
+
+> https://quay.io/repository/submariner/lighthouse-controller?tab=tags
+> https://quay.io/repository/submariner/lighthouse-coredns?tab=tags
+
+<!-- TODO: Aswin: the openshift-coredns part -->
+
+# Step 3: update the operator version references and create a release
+
+Once the other builds have finished and you have 0.2.0-rc0 release tags for the submariner and lighthouse projects, you can proceed with changes to the operator.
+
+## Change referenced versions
+
+Edit the operator [versions](https://github.com/submariner-io/submariner-operator/edit/master/pkg/versions/versions.go) file and change the project version constants to reference the new release, "0.2.0-rc0".
+
+> https://github.com/submariner-io/submariner-operator/edit/master/pkg/versions/versions.go
+
+Create a pull request, wait for the CI job to pass, and get approval/merge. See an example PR [here](https://github.com/submariner-io/submariner-operator/pull/276)
+
+
+## Create a submariner-operator release
+
+Assuming you have an existing submariner-operator git directory, run the following steps:
+
+```bash
+cd submariner-operator
+git remote add upstream ssh://git@github.com/submariner-io/submariner-operator
+git fetch -a -v -t
+git checkout remotes/upstream/master -B master
+git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
+git push upstream v0.2.0-rc0
+```
+
+
+A tagged release should appear [here](https://github.com/submariner-io/submariner-operator/tags).
+
+> https://github.com/submariner-io/submariner-operator/tags
+
+A build for v0.2.0-rc0 should start and appear under the under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/submariner-operator/branches).
+
+> https://travis-ci.com/github/submariner-io/submariner-operator/branches
+
+Look under the "Active branches" section for v0.2.0-rc0 , and monitor the build.
+Verify that the build successfully completes as indicated by a green checkmark at the right.
+At this point the images are pushed to quay.io.
+
+Finally once that has finished, a 0.2.0-rc0 tag will be available [here](https://quay.io/repository/submariner/submariner-operator?tab=tags).
+
+> https://quay.io/repository/submariner/submariner-operator?tab=tags
```

Comment:
L114-116 can be removed.

Reply:
Opinion, I really prefer how it looks with this, it servers as visual reference, and provides the ability to see the URL easily without need to click

Reply:
OK we can leave the link but L110 is redundant with L108.

Reply:
True

Reply:
Done